HeterogeneousCore/SonicTriton/test/tritonTest_cfg.py

```diff
@@ -62,6 +80,21 @@
                 modelName = cms.string(model),
                 modelVersion = cms.string(""),
                 modelConfigPath = cms.FileInPath("HeterogeneousCore/SonicTriton/data/models/{}/config.pbtxt".format(model)),
+                verbose = cms.untracked.bool(options.verbose or options.verboseClient),
+                useSharedMemory = cms.untracked.bool(not options.noShm),
+                compression = cms.untracked.string(options.compression),
+                Retry = (
+                  cms.VPSet(
+                    cms.PSet(
+                      retryType = cms.string('RetrySameServerAction'),
+                      allowedTries = cms.untracked.uint32(options.tries)
+                    )
+                  ) if options.retryAction == 'same' else cms.VPSet(
+                    cms.PSet(
+                      retryType = cms.string('RetryActionDiffServer')
+                    )
+                  )
+                )
```

Comment:
this should eventually be absorbed into `HeterogeneousCore/SonicTriton/python/customize.py`

Reply:
moved like this: https://github.com/fastmachinelearning/cmssw/pull/24/commits/7dc9e7162d1fb97fdd6e594b7541c67652ac7e02